sequencing/ATACSeq/preprocessing/progressReport.sh

```diff
@@ -134,10 +129,17 @@ do
     fi
     
     if [ ! -s ${PEAKDIR}/MACS/BAMPE/${sampleName}.broadPeak.filt ]
+    then
+        echo -n "N" >> ${METADIR}/summariseSampleProcessingProgress.csv
+    else
+        echo -n "Y," >> ${METADIR}/summariseSampleProcessingProgress.csv
+    fi
+    
+    if [ ! -s ${PEAKDIR}/HMMRATAC/${sampleName}.sorted.filteredPeaks.gappedPeak ]
     then
         echo "N" >> ${METADIR}/summariseSampleProcessingProgress.csv
     else
-        echo "Y" >> ${METADIR}/summariseSampleProcessingProgress.csv
+        echo "Y," >> ${METADIR}/summariseSampleProcessingProgress.csv
```

Comment:
This is the final column of the csv file, why does this have a comma after the Y?